com.snipaste.Snipaste.yaml

```diff
@@ -0,0 +1,42 @@
+app-id: com.snipaste.Snipaste
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: snipaste
+separate-locales: false
+
+finish-args:
+  - --device=all
+
+  - --socket=wayland
+  - --socket=fallback-x11
+
+  - --share=ipc
+  - --share=network
+
+  - --filesystem=xdg-pictures
+  - --filesystem=xdg-run/pipewire-0
+
+  - --env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons
+
+  # D-Bus Access
+  - --talk-name=org.freedesktop.Notifications
```

Comment:
if it uses libnotify for notifications and libnotify 0.8 is present in the appimage this won't be needed.

Reply:
removed this dbus access